packages/lumx-react/src/components/date-picker/DatePickerControlled.tsx

```diff
@@ -53,13 +54,25 @@ export const DatePickerControlled: Comp<DatePickerControlledProps, HTMLDivElemen
         return getMonthCalendar(localeObj, selectedMonth, minDate, maxDate);
     }, [locale, minDate, maxDate, selectedMonth]);
 
+    // Only activate aria-live after the month has changed
+    const [labelAriaLive, setLabelAriaLive] = React.useState<'polite'>();
+    const prevSelectedMonthRef = React.useRef<Date>();
+    React.useEffect(() => {
+        const prevSelectedMonth = prevSelectedMonthRef.current;
+        if (prevSelectedMonth && !isSameDay(prevSelectedMonth, selectedMonth)) {
+            setLabelAriaLive('polite');
+        }
+        prevSelectedMonthRef.current = selectedMonth;
+    }, [selectedMonth]);
```

Comment:
Try to avoid this with aria-atomic (make sure the mont-year is not announced when opening the date picker dialog)

Reply:
Can't make it work (at least on VoiceOver)

Reply:
Mabe we could add a little comment to explain why we do this this way (ie. the aria live is reading the text on picker opening)